libs/ledger-live-common/src/families/ethereum/transaction.ts

```diff
@@ -154,7 +177,15 @@ export const fromTransactionRaw = (tr: TransactionRaw): Transaction => {
     // FIXME this is not good.. we're dereferencing here. we should instead store an index (to lookup in currency.units on UI)
     networkInfo: networkInfo && {
       family: networkInfo.family,
-      gasPrice: fromRangeRaw(networkInfo.gasPrice),
+      gasPrice: networkInfo.gasPrice
+        ? fromRangeRaw(networkInfo.gasPrice)
+        : undefined,
```

Comment:
just a detail but it would make more sense to use `null` here as the variable is actually defined but has `null` value